.claude/hooks/auto-handoff-stop.py

```
@@ -9,12 +14,14 @@
-        pct = int(open(ctx_files[0]).read().strip())
+        with open(ctx_files[0]) as f:
+            pct = int(f.read().strip())
         if pct >= 85:
             print(json.dumps({
                 "decision": "block",
                 "reason": f"Context at {pct}%. Run: /create_handoff"
             }))
             sys.exit(0)
-    except: pass
+    except Exception:
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)